WordPress/Classes/ViewRelated/Reader/Comments/ReaderCommentsNotificationSheetViewController.swift

```diff
@@ -154,103 +151,48 @@ import WordPressUI
+        static var contentInsets = UIEdgeInsets(.all, 20)
         static var switchContainerContentSpacing: CGFloat = 4
         static var switchLabelVerticalPadding: CGFloat = 6
         static var switchButtonTrailingPadding: CGFloat = 2
-        static var iPadAdditionalBottomPadding: CGFloat = 5
-    }
-
-    /// Returns the vertical padding outside the intrinsic height of the `containerStackView`, so the component is displayed properly.
-    var verticalPadding: CGFloat {
-        return Constants.contentInsets.top
-            + Constants.contentInsets.bottom
-            + additionalVerticalPadding
-    }
-
-    /// Calculates the default top margin from the `BottomSheetViewController`, plus the bottom safe area inset.
-    /// The 5pt is for an extra bottom padding on iPad, to make it look better.
-    var additionalVerticalPadding: CGFloat {
-        WPDeviceIdentification.isiPad() ? Constants.iPadAdditionalBottomPadding
-            : BottomSheetViewController.Constants.additionalContentTopMargin + view.safeAreaInsets.bottom
     }
 
     func configureViews() {
         view.addSubview(scrollView)
-        view.pinSubviewToAllEdges(scrollView, insets: Constants.contentInsets)
+        scrollView.pinEdges(insets: Constants.contentInsets)
 
         // don't update the content size at this state, because the layout pass has not completed.
         // doing so will cause the height to be incorrectly assigned to the preferredContentSize.
         updateViews(updatesContentSize: false)
     }
 
     func updateViews(updatesContentSize: Bool) {
-        descriptionLabel.setText(isNotificationEnabled ? .descriptionTextForEnabledNotifications : .descriptionTextForDisabledNotifications)
+        descriptionLabel.setText(isNotificationEnabled ? Strings.descriptionTextForEnabledNotifications : Strings.descriptionTextForDisabledNotifications)
         switchButton.isOn = isNotificationEnabled
 
         if updatesContentSize {
             view.layoutIfNeeded()
             updatePreferredContentSize()
         }
-
-        // readjust drawer height on content size changes.
-        if let drawer = presentedVC {
-            drawer.transition(to: drawer.currentPosition)
-        }
     }
 
-    func updatePreferredContentSize() {
+    private func updatePreferredContentSize() {
+        let verticalPadding = Constants.contentInsets.top + Constants.contentInsets.bottom
         preferredContentSize = CGSize(width: preferredContentSize.width, height: scrollView.contentSize.height + verticalPadding)
     }
 
-    func toggleNoticeLock(_ locked: Bool) {
-        // only enable locking/unlocking notices on iPhone. Notices should always be shown in iPad since it's displayed in a popover view.
-        guard WPDeviceIdentification.isiPhone() else {
-            return
-        }
-
-        ActionDispatcher.dispatch(locked ? NoticeAction.lock : NoticeAction.unlock)
```

Comment:
I thought it was safer to just remove it, including the `WPDeviceIdentification.isiPhone()` and reliance on dismiss callbacks. I don't see a big deal showing a notice here.